lib/DDG/Spice/Rfc.pm

```diff
@@ -0,0 +1,29 @@
+package DDG::Spice::Rfc;
+
+# ABSTRACT: Search for Request for Comments documents.
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1; # only enable for non-JSONP APIs (i.e. no &callback= parameter)
+
+# API endpoint - http://docs.duckduckhack.com/walkthroughs/forum-lookup.html#api-endpoint
+spice to => 'https://rfcsearch-gorun.rhcloud.com/?q=$1';
```

Comment:
@moollaza Yes but the first response was short, pointing me to a (quite large) XML index of every RFC. Not exactly what I had in mind so I explained the use case for having search results in JSON format . Still waiting to hear back from them concerning the API and their TOS.